packages/icons/scripts/svgProcess.js

```diff
@@ -0,0 +1,33 @@
+const fs = require('fs');
+const path = require('path');
+
+const srcDir = `${__dirname}/../src/svg`;
```

Comment:
Please use `resolve(__dirname, ...)` as is for example in https://github.com/lmc-eu/spirit-design-system/blob/main/packages/web/scripts/prepareDist.js#L10 to resolve an absolute path.